boltdesignsystem/bolt
# Bolt band in modal fix

## Jira

http://vjira2:8080/browse/WWWD-4249

## Summary

When we use a bolt band with modal and a lot of content was added to this modal in mobile Safari top of the content was cut.

## Details

I update styles in bolt band and compare is are some differences before and after the change.
![compare-band-versions](https://user-images.githubusercontent.com/16049049/69630479-4555d280-104d-11ea-97f8-73a014bd3b3b.jpg)
Nothing changes after the update but i must update band snapshots to pass new styling.
Right now in Safari and other browsers bands in modal looks good.

## How to test

Check if all test is passed and if this page https://boltdesignsystem-rjqreb3m5.now.sh/pattern-lab/patterns/02-components-modal-50-modal-usage-content/02-components-modal-50-modal-usage-content.html is displayed correctly in Safari and other browsers in mobile view.
